intel-sgx/pcs/src/tcb_info.rs

```diff
@@ -221,7 +221,7 @@ impl TcbData<Unverified> {
 }
 
 impl<V: VerificationType> TcbData<V> {
-    pub(crate) fn tcb_levels(&self) -> &Vec<TcbLevel> {
+    pub fn tcb_levels(&self) -> &Vec<TcbLevel> {
```

Comment:
Can you return an iterator over the cpusvn and pcesvn for each level? The whole point of using type states here is to prevent us from accessing the TCB levels without checking whether the TcbInfo is valid in the first place. Making this function public, completely breaks that logic.

Reply:
The intention to keep these private should be documented.